### ISSUE TITLE
Fuzzy search

### ISSUE THREAD

**Issue by noodlebreak** [DoTheEvo/ANGRYsearch#14]
Hi,

This is absolutely great, almost like Everything.
But it's lacking fuzzy search.

**Comment by DoTheEvo** [comment]
IMO would not work well, thinking about it I see two problems
* it would be slow as hell, no indexing to help, its going through raw data, all of them to be any useful and then its some ranking to get the closest matches up top...
* results might be overwhelming and mostly useless

**Comment by sammoth** [comment]
I don't think it would have to be slow as hell. I use `fzf` for a similar purpose, and with around 3 million paths it is still very responsive, taking less than half a second to update results.
